website/next.config.js

```diff
@@ -37,6 +37,13 @@ export default withGuildDocs({
       '/docs/testing': '/docs/features/testing',
       '/docs/integrations': '/docs',
       '/docs/features': '/docs',
+      '/v2/features/health-check': '/v2',
```

Comment:
why do we just point to the `/v2` URL? we don't have a dedicated page for these?

Reply:
We dont pages for those subjects in V2.
@dotansimha